openedx/core/djangoapps/user_api/accounts/api.py

```diff
@@ -331,35 +331,6 @@ def _send_email_change_requests_if_needed(data, user):
             )
 
 
-@helpers.intercept_errors(errors.UserAPIInternalError, ignore_errors=[errors.UserAPIRequestError])
-def activate_account(activation_key):
-    """Activate a user's account.
-
-    Args:
-        activation_key (unicode): The activation key the user received via email.
-
-    Returns:
-        None
-
-    Raises:
-        errors.UserNotAuthorized
-        errors.UserAPIInternalError: the operation failed due to an unexpected error.
-
-    """
-    # TODO: Confirm this `activate_account` is only used for tests. If so, this should not be used for tests, and we
-    # should instead use the `activate_account` used for /activate.
-    set_custom_metric('user_api_activate_account', 'True')
```

Comment:
**Note to Reviewers:** Thanks to @robrap's metric here, we've verified that `activate_account` is used only in tests and not in production.